webapp/services.py

```diff
@@ -30,20 +30,22 @@ def find_all_assets(self):
 
     def find_assets(
         self,
-        tag: str = "abc",
-        asset_type: str = "image",
-        product_types: list = ["a", "b"],
-        author_email: str = "abc@g.com",
-        start_date: str = "2024-01-01",
-        end_date: str = "2024-10-14",
-        salesforce_campaign_id: str = "1234",
-        language: str = "English",
+        tag: str = "",
+        asset_type: str = "",
+        product_types: list = [],
+        author_email: str = "",
+        start_date: str = "",
+        end_date: str = "",
+        salesforce_campaign_id: str = "",
+        language: str = "",
         page=1,
         per_page=6,
         order_by=Asset.created,
         desc_order=True,
         include_deprecated=False,
-        file_types: list = ["a", "b"],
+        file_types: list = [],
```

Comment:
better to use an immutable type as a default
```suggestion
        file_types: list = (),
```